app/views/admin/services/editors/_labels.html.erb

```diff
@@ -6,7 +6,7 @@
     </div>
 
     <script>
-        __LABELS__=<%= raw ActsAsTaggableOn::Tag.most_used.to_json %>
+        __LABELS__=<%= raw ActsAsTaggableOn::Tag.distinct.to_json %>
```

Comment:
I know this is how the plugin works to do the typeahead but imagine if there were 2000 tags thats a lot to load into an object on the page. Ideally this would be an API call.

Reply:
By default fetching all the tags won't bring up dupes, so we can do:

```suggestion
        __LABELS__=<%= raw ActsAsTaggableOn::Tag.all.to_json %>
```

Agreed it there are loads it might become an issue! I've just had a look in Metabase, at the moment we've got ~90 tags in total so hopefully it should be okay for now.